internal/ingress/annotations/annotations.go

```diff
@@ -76,6 +77,7 @@ type Ingress struct {
 	ExternalAuth         authreq.Config
 	HealthCheck          healthcheck.Config
 	Proxy                proxy.Config
+	ProxyProtocol        bool
```

Comment:
If this is specific to SSL passthrough, I'd rather have it grouped together with the `ssl-passthrough` annotation.

Reply:
@antoineco How it should be grouped?

As I understand a lot of users require SSL Passthrough without Proxy Protocol. And ssl-passthrough option is boolean.